tests/MultipleInputParamTest.php

```
@@ -0,0 +1,65 @@
+            'category' => 'electronics',
+            'page' => '2',
+            'limit' => '10',
+        ];
+
+        $args = $this->namedParameter->getParameters([$this->multipleInputTest, 'onGet'], $params);
+        /** @phpstan-ignore-next-line Parameter type mismatch, runtime provides correct types */
+        $ro = $this->multipleInputTest->onGet(...array_values($args));
+        assert(is_array($ro->body));
+
+        $this->assertSame(200, $ro->code);
+        $this->assertSame('laptop', $ro->body['search']['query']);
```

PHPStan (2.1.15)

Cannot access offset 'query' on mixed.


(offsetAccess.nonOffsetAccessible)